### PR TITLE
PEPPER-324: [DSS] PS: Text Updates to Homepage, Dashboard, and FAQ

### PR DESCRIPTION
**Pull request by walejand**

### CODE REVIEW

File: ddp-workspace/projects/ddp-singular/src/assets/i18n/en.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
           {
             "Name": "How will Project Singular learn about single ventricle heart defects?",
             "Answers": [
-              "We will collect a saliva (spit) sample or a cheek swab from people enrolled in this study. We will use these samples to look at DNA. We want to see if we can find the causes of single ventricle heart defects. If we understand the causes, researchers may be able to develop new and better treatments.",
+              "We will collect a saliva sample or a cheek swab from people enrolled in this study. We will use these samples to look at DNA. We want to see if we can find the causes of single ventricle heart defects. If we understand the causes, researchers may be able to develop new and better treatments.",
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-singular/src/assets/i18n/en.json]: It's written in the ticket "[take out the mention of cheek swab"](https://broadworkbench.atlassian.net/browse/PEPPER-324#:~:text=Change%20language%20of%201st%20sentence%20in%20answer%20for%20FAQ%20question%20%22How%20will%20Project%20Singular%20learn%20about%20single%20ventricle%20heart%20defects%3F%22%20to%20say%20%22We%20will%20collect%20a%20saliva%20sample%20from%20people%20enrolled%20in%20the%20study%22%2C%20taking%20out%20the%20word%20%22spit%22%20and%20the%20mention%20of%20cheek%20swab.).
Thus, the first sentence should be 
`We will collect a saliva sample from people enrolled in this study.`

File: ddp-workspace/projects/ddp-singular/src/assets/i18n/en.json [hunk 2]
```diff
@@ -108,8 +108,8 @@
           {
             "Name": "What should I know about participating in this study?",
             "Answers": [
-              "You won’t need to go to a clinic to take part. You can sign up on this website. We will mail you a saliva (spit) or cheek swab collection kit to use at home.",
-              "Most participants will receive a saliva (spit) collection kit.  Babies and toddlers under the age of 3 will receive a cheek swab kit.",
+              "You won’t need to go to a clinic to take part. You can sign up on this website. We will mail you a saliva or cheek swab collection kit to use at home.",
+              "Most participants will receive a saliva collection kit.  Babies and toddlers under the age of 3 will receive a cheek swab kit.",
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-singular/src/assets/i18n/en.json]: 1. first paragraph should be  
`"You won’t need to go to a clinic to take part. You can sign up on this website. We will mail you a saliva collection kit to use at home."`

see [link](https://broadworkbench.atlassian.net/browse/PEPPER-324#:~:text=Change%20language%20in%20the%203rd%20sentence%20of%20the%201st%20bulleted%20answer%20for%20FAQ%20question%20%22What%20should%20I%20know%20about%20participating%20in%20this%20study%3F%22%20to%20say%20%22We%20will%20mail%20you%20a%20saliva%20collection%20kit%20to%20use%20at%20home.%22%2C%20taking%20out%20the%20word%20%22spit%22%20and%20the%20mention%20of%20cheek%20swab.)

Comment (alextanlan) [reply ddp-workspace/projects/ddp-singular/src/assets/i18n/en.json]: 2. The second paragraph ("Most participants ...") should be deleted.

see [the ticket](https://broadworkbench.atlassian.net/browse/PEPPER-324#:~:text=Delete%20the%20entirety%20of%20the%202nd%20bulleted%20answer%20for%20the%20same%20question%20to%20remove%20%22Most%20participants%20will%20receive%20a%20saliva%20(spit)%20collection%20kit.%20Babies%20and%20toddlers%20under%20the%20age%20of%203%20will%20receive%20a%20cheek%20swab%20kit%22.)

Comment (walejand) [reply ddp-workspace/projects/ddp-singular/src/assets/i18n/en.json]: Thanks for feedback :), I have already updated with your comments

**Review by alextanlan — APPROVED** [review]

**Review by juanseba21 — APPROVED** [review]

**Pull request merged**